lib/appmap/metadata.rb

```diff
@@ -9,20 +9,21 @@ def detect
         {
           app: AppMap.configuration.name,
           language: {
-            name: 'ruby',
+            name: "ruby",
             engine: RUBY_ENGINE,
             version: RUBY_VERSION
           },
           client: {
-            name: 'appmap',
+            name: "appmap",
             url: AppMap::URL,
             version: AppMap::VERSION
-          }
+          },
+          timestamp: Time.now.to_f
```

Comment:
ISO timestamp would be much more useful.

Reply:
Why? It’s the same information.

Reply:
Because numeric timestamps are utterly meaningless to a human :)